yb-voyager/cmd/importData.go

```diff
@@ -941,27 +927,35 @@ func executeSqlFile(file string, objType string) {
 		}
 	}()
 
-	var errOccured = 0
 	sqlInfoArr := createSqlStrInfoArray(file, objType)
 	for _, sqlInfo := range sqlInfoArr {
 		if conn == nil {
 			conn = newTargetConn()
 		}
+
+		setOrSelectStmt := strings.HasPrefix(sqlInfo.stmt, "SET ") || strings.HasPrefix(sqlInfo.stmt, "SELECT ")
```

Comment:
can we make the string comparison case insensitive here also as asked at other place.

Reply:
Done.